@@ -45,12 +45,14 @@ wqy-microhei-fonts          # a compact
 -@input-methods
 -scim*
 -m17n*
--ibus*
 -iok
 
 # Fix https://bugzilla.redhat.com/show_bug.cgi?id=1429132
 # Why is this not pulled in by anaconda???
 storaged
 
+# fspin custom
+kernel-devel
+
 %end
 
diff -up ./fedora-mate-common.ks.orig ./fedora-mate-common.ks
--- ./fedora-mate-common.ks.orig        2018-10-17 21:03:49.000000000 +0000
+++ ./fedora-mate-common.ks     2019-04-17 03:26:28.516273893 +0000
@@ -51,4 +51,9 @@ nss-mdns
 # Legacy cmdline things we don't want
 -telnet
 
+# fspin custom
+kernel-devel
+java-11-openjdk-headless
+java-openjdk-headless
+
 %end
diff -up ./fedora-workstation-common.ks.orig ./fedora-workstation-common.ks
--- ./fedora-workstation-common.ks.orig 2018-10-17 21:03:49.000000000 +0000
+++ ./fedora-workstation-common.ks      2019-04-17 03:26:42.720334257 +0000
@@ -24,4 +24,9 @@
 -gfs2-utils
 -reiserfs-utils
 
+# fspin custom
+kernel-devel
+java-11-openjdk-headless
+java-openjdk-headless
+
 %end
diff -up ./fedora-xfce-common.ks.orig ./fedora-xfce-common.ks
--- ./fedora-xfce-common.ks.orig        2018-10-17 21:03:49.000000000 +0000
+++ ./fedora-xfce-common.ks     2019-04-17 03:24:25.550107496 +0000
@@ -39,4 +39,7 @@ system-config-printer
 -aspell-*                   # dictionaries are big
 -xfce4-sensors-plugin
 
+# fspin custom
+kernel-devel
+
 %end
